README.md

```diff
@@ -1,366 +1,542 @@
+the folder recursively (including only the resulting `.c4gh` files) under
+`<new_folder_name>`.
 
-**Notes**: The tool calls the [encrypt](#Encrypt) module internally, therefore similar behavior to that command is expected, including the creation of hash files. In addition,
+**Notes**:
 
-- For encryption with [multiple public keys](<#Encrypt-file(s)-with-multiple-keys>), concatenate all public keys into one file and pass it as the argument to `encrypt-with-key`.
-- If the input includes encrypted files, the tool will exit without performing further tasks.
-- The encrypted files will be created next to their unencrypted counterparts.
-- The tool will not overwrite existing encrypted files. It will exit early if encrypted counterparts of the source files already exist with the same source path.
-- If the flag `--force-overwrite` is used, the tool will overwrite any already existing file.
-- The cli will exit if the input has any un-encrypred files. To override that, use the flag `--force-unencrypted`.
+- The command internally calls the [encrypt](#Encrypt) module when performing
+encrypt-on-upload, mirroring its behavior, including the creation of hash files.
+- For encryption with [multiple public keys](#Encrypt-files-with-multiple-keys),
+concatenate all public keys into a single file and provide it using the
+`-encrypt-with-key` flag.
+- If the input includes already encrypted files, the process will exit without
+further processing.
+- Encrypted files are created in the same directory as their unencrypted counterparts.
+- If encrypted counterparts of the input files already exist, the process will
+exit without further processing.
+- If a file has already been uploaded, the process will exit without further
+processing. To overwrite existing files, use the `-force-overwrite` flag.
 
 ## List
 
-All the following cases require a [configuration file to be downloaded](https://github.com/NBISweden/sda-cli/blob/documentation/datset-list/README.md#download-the-configuration-file) before starting the operation, and the `list` parameter should be used.
+Before using the `list` functionality, ensure you have [downloaded the configuration file](#download-the-configuration-file).
 
 ### List uploaded files
 
-This feature returns all the files in the user's inbox recursively and can be executed using:
+You can retrieves all the files along with their sizes in the user's inbox,
+including those in subdirectories, recursively using the following command:
 
 ```bash
 ./sda-cli [-config <configuration_file>] list
 ```
 
-It also allows for requesting files/filepaths with a specified prefix using:
+You can also list files or file paths with a specific prefix by using:
 
 ```bash
 ./sda-cli [-config <configuration_file>] list <prefix>
 ```
 
-This command will return any file/path starting with the defined `<prefix>`.
-If no config is given by the user, the tool will look for a previous login from the user.
```

Comment:
Just wondering, do we have the above behavior as a feature?

Reply:
`GetAuth` will look for the folder `.sda-cli-session` for the configuration file. But since the `login` function is not working, we should not mention it here. I will remove it.

Reply:
yes, I think that makes sense